5-app-infra/README.md

```diff
@@ -0,0 +1,41 @@
+# 5-app-infra
+
+The purpose of this step is to set up an example app, deploying a Compute Engine instance in one of the business unit created on step 4-projects.
+This Compute Engine instance will be created using the base network created during step 3-networks to acess private services.
+
```

Comment:
Can we add a short blurb on how deployment in 5-app-infra in general works?
i.e we push to repo created in 4-projects, CB SA can impersonate project SA based on https://github.com/terraform-google-modules/terraform-example-foundation/blob/3c62c9e47961142ecd0d67ff6dea4e153cb541fc/4-projects/business_unit_1/development/example_base_shared_vpc_project.tf#L31-L32 etc